e2e_tests/tests/command/test_shell.py

```diff
@@ -3,8 +3,10 @@
 import pytest
 
 import determined as det
+from tests import api_utils
 from tests import command as cmd
-from tests.cluster.test_users import det_spawn
+from tests.cluster.test_users import det_spawn, logged_in_user
+from tests.config import ADMIN_CREDENTIALS
```

Comment:
do not import symbols directly; import the module and use the module name when you reference the symbol.

So for `from tests.cluster.test_users import symbol1, symbol2` you should:
```python
from tests.cluster import test_users
...
test_users.det_spawn()
test_users.logged_in_user()
```

That implies you'll have to fix some existing code, which is fine.  This falls under the [google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports), which we try to stick to.  (you'll find lots of e2e_tests code which is sloppy about this... we're trying to get better).

Also, for some reason, we always import `tests.config` like:
```python
from tests import config as conf`
```

which I think is silly but I don't see a point in varying from the standard pattern for this file.  So that's how you should do that particular import.

Reply:
Gotchya, changed!